src/status_im2/contexts/onboarding/notifications/views.cljs

```diff
@@ -0,0 +1,71 @@
+(ns status-im2.contexts.onboarding.notifications.views
+  (:require
+    [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]
+    [react-native.core :as rn]
+    [react-native.platform :as platform]
+    [status-im.notifications.core :as notifications]
+    [status-im2.contexts.onboarding.notifications.style :as style]
+    [status-im2.contexts.onboarding.common.background :as background]))
+
+(defn page-title
+  []
+  [rn/view {:style style/title-container}
+   [quo/text
+    (merge {:style {:color colors/white}}
+           style/title-text-style)
+    (i18n/label :t/intro-wizard-title6)]
+   [quo/text
+    (merge {:style {:color colors/white}}
+           style/subtitle-text-style)
+    (i18n/label :t/enable-notifications-sub-title)]])
+
+(defn navigate-back
+  [root]
+  [rn/view {:margin-top (if platform/ios? 44 0)}
+   [quo/page-nav
+    (merge {:horizontal-description? false
+            :one-icon-align-left?    true
+            :align-mid?              false
+            :page-nav-color          :transparent
+            :mid-section             {:type            :text-with-description
+                                      :main-text       nil
+                                      :description-img nil}
+            :left-section            {:icon                  :i/arrow-left
+                                      :icon-background-color colors/white-opa-5
+                                      :icon-override-theme   :dark
+                                      :type                  :shell
+                                      :on-press              #(rf/dispatch [:init-root root])}})]])
+
+(defn enable-notification-buttons
+  []
+  [rn/view {:style style/enable-notifications-buttons}
+   [quo/button
+    {:on-press                  #(do (rf/dispatch [::notifications/switch true platform/ios?])
```

Comment:
If you can avoid @jo-mut, try to use `(fn [] ...)` on multiple expressions instead of `#(do ...`). It's in the style guide.